src/services/findAllReferences.ts

```diff
@@ -1837,6 +1837,9 @@ export namespace Core {
                 }
                 // falls through I guess
             case SyntaxKind.Identifier:
+                if (node.flags & NodeFlags.JSDoc && !isInJSFile(node)) {

```

Comment:
I need to additionally check if this is in an "any type tag" as those should be ignored whereas nodes in semantic~ jsdoc tags shouldn't (like in `@see`, `@link`, etc). I couldn't find any utility that would check something like this - I could create such and manually curate the list of those tags in it.

But there is another question: should those references in type-like tags always be ignored? Or should TS try to resolve corresponding TS types? There is always a risk that those won't match but in such a case displaying `any` (likely as a result of displaying `errorType`) could actually be OK.

An alternative fix for the crashes would be to ignore symbol-less declarations for now but I intentionally didn't go that route. It seemed like a bandaid for a deeper problem and I think handling this at a higher level would be better. I'm just not sure now what's the exact desired behavior.

cc @sandersn

Reply:
I think you're asking whether find-all-refs should skip the references in `@see` et al? Is that right? Ideally the answer is No. Symbol resolution for those tags works outside the normal name resolution code, as I recall, via resolveJSDocMemberName. That function is called from getSymbolAtLocation, the usual entrypoint for public users of the API. You might want to call one of those two functions.

A utility to identify tags with semantics in Typescript would be useful, and I don't think it exists yet. scanner.ts does have a regex, but that's not useful for a parse tree. (and, besides, `@deprecated` should be included in the post-parser predicate, but not the scanner one.)

Please add a test either way.